test/pdfboxing/common_test.clj

```diff
@@ -10,18 +12,28 @@
 
 (deftest obtain-document-returns-pddocument-if-provided-string-check
   (testing "obtain-document returns a PDDocument if a string path to a PDF is a supplied"
-    (is (true?  (instance?
-                   org.apache.pdfbox.pdmodel.PDDocument
-                   (obtain-document "test/pdfs/hello.pdf"))))))
+    (let [doc (obtain-document "test/pdfs/hello.pdf")]
+      (try
+        (is (true? (instance? PDDocument doc)))
+        (finally
+          (when (instance? PDDocument doc)
+            (.close doc)))))))
 
 (deftest obtain-document-returns-pddocument-if-provided-pddocument-check
   (testing "obtain-document returns a PDDocument if PDDocument is provided"
-    (is (true?  (instance?
-                   org.apache.pdfbox.pdmodel.PDDocument
-                   (obtain-document (obtain-document "test/pdfs/hello.pdf")))))))
+    (let [doc1 (obtain-document "test/pdfs/hello.pdf")
+          doc2 (obtain-document doc1)]
```

Comment:
It seemed that you wanted to test that if you nested `obtain-document` calls you would still receive the original document. Your comment states, "obtain-document returns a PDDocument if PDDocument is provided." I was just fixing the case that neither document was closed.